src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -546,7 +565,7 @@ class MessageSection extends Component {
                 <ul
                   className='message-list'
                   ref={(c) => { this.messageList = c; }}
-                  style={{background:pane}}>
+                  style={messageBackgroundStyles}>
```

Comment:
Let's have the `background-repeat:none` property as well to the styles. @madhavrathi